### PR TITLE
review: src/guide/essentials/reactivity-fundamentals.md and template-syntax.md

### PR DESCRIPTION
**Pull request by pakchoily**
All modified expressions are based on the latest English documents,just inspect for composition api preference
## Additional Information
- commit [update: multiple translation verification](https://github.com/vuejs-translations/docs-zh-cn/commit/7d12c49596d3ddd9284f581bd6629e9858ff69b5)
    - remain property | Vue | state
    - line217 modify expression,translate ' tick'
    - line307|344|363|368 “原始” seems to understand better
    - line158|317|319|355|378 modify expression
    - line428 fix punctuation
    - line454|468|478 modify expression based on English doc, “顶级” be better
    - line504|519 important correction expression

### CODE REVIEW

File: src/guide/essentials/reactivity-fundamentals.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ export default {
 
 ### DOM 更新时机 {#dom-update-timing}
 
-当你更改响应式状态后，DOM 也会自动更新。然而，你得注意 DOM 的更新并不是同步的。相反，无论你改变了多少个状态，Vue 都会将它们推入更新循环的 “下个 tick” 执行以确保每个需要更新的组件都只会更新一次。
+当你更改响应式状态后，DOM 也会自动更新。然而，你得注意 DOM 的更新并不是同步的。相反，Vue将缓冲它们直到更新周期的 “下个时机” 以确保无论你进行了多少次声明更改，每个组件都只需要更新一次。
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:217]:
```suggestion
当你更改响应式状态后，DOM 也会自动更新。然而，你得注意 DOM 的更新并不是同步的。相反，Vue 将缓冲它们直到更新周期的 “下个时机” 以确保无论你进行了多少次声明更改，每个组件都只需要更新一次。
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 2]
```diff
@@ -341,9 +341,9 @@ console.log(proxy.nested === raw) // false
 
 `reactive()` API 有两条限制：
 
-1. 仅对对象类型有效（对象、数组和 `Map`、`Set` 这样的[集合类型](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects#%E4%BD%BF%E7%94%A8%E9%94%AE%E7%9A%84%E9%9B%86%E5%90%88%E5%AF%B9%E8%B1%A1)），而对 `string`、`number` 和 `boolean` 这样的 [基础类型](https://developer.mozilla.org/zh-CN/docs/Glossary/Primitive) 无效。
+1. 仅对对象类型有效（对象、数组和 `Map`、`Set` 这样的[集合类型](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects#%E4%BD%BF%E7%94%A8%E9%94%AE%E7%9A%84%E9%9B%86%E5%90%88%E5%AF%B9%E8%B1%A1)），而对 `string`、`number` 和 `boolean` 这样的 [原始类型](https://developer.mozilla.org/zh-CN/docs/Glossary/Primitive) 无效。
 
-2. 因为 Vue 的响应式系统是通过属性访问进行追踪的，因此我们必须始终保持对该响应式对象的引用。这意味着我们不可以随意地 “替换” 一个响应式对象：
+2. 因为 Vue 的响应式系统是通过property访问进行追踪的，因此我们必须始终保持对该响应式对象的相同引用。这意味着我们不可以随意地 “替换” 一个响应式对象：
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:346]:
```suggestion
2. 因为 Vue 的响应式系统是通过 property 访问进行追踪的，因此我们必须始终保持对该响应式对象的相同引用。这意味着我们不可以随意地 “替换” 一个响应式对象：
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 3]
```diff
@@ -352,20 +352,20 @@ console.log(proxy.nested === raw) // false
    state = reactive({ count: 1 })
    ```
 
-   同时这也意味着把响应式对象的某个基础类型属性传入函数，或是从响应式对象中解构属性时，我们会失去响应性：
+   同时这也意味着当指定或解构一个响应式对象的property为局部变量，或者把property传入一个方法，我们都会失去响应性连接：
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:355]:
```suggestion
   同时这也意味着当指定或解构一个响应式对象的 property 为局部变量，或者把 property 传入一个方法，我们都会失去响应性连接：
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 4]
```diff
@@ -375,15 +375,15 @@ console.log(proxy.nested === raw) // false
 
 ## `ref()` 定义响应式变量 \*\* {#reactive-variables-with-ref}
 
-为了解决 `reactive()` 带来的限制，我们提供了另一个 [`ref()`](/api/reactivity-core.html#ref) 方法来帮我们创建响应式的 **ref**，它可以装载任何值类型：
+为了解决 `reactive()` 带来的限制，Vue也提供了一个 [`ref()`](/api/reactivity-core.html#ref) 方法来允许我们创建可以使用任何值类型的响应式 **ref**：
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:378]:
```suggestion
为了解决 `reactive()` 带来的限制，Vue 也提供了一个 [`ref()`](/api/reactivity-core.html#ref) 方法来允许我们创建可以使用任何值类型的响应式 **ref**：
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 5]
```diff
@@ -375,15 +375,15 @@ console.log(proxy.nested === raw) // false
...
 
 ```js
 import { ref } from 'vue'
 
 const count = ref(0)
 ```
 
-`ref()` 从参数中获取到值，将其包装为一个带 `.value` 属性的对象：
+`ref()` 从参数中获取到值，将其包装为一个带 `.value` property的ref对象：
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:386]:
```suggestion
`ref()` 从参数中获取到值，将其包装为一个带 `.value` property 的 ref 对象：
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 6]
```diff
@@ -395,9 +395,9 @@ count.value++
 console.log(count.value) // 1
 ```
 
-你也可以看看：[为 ref 标注类型](/guide/typescript/composition-api.html#typing-ref)。 <sup class="vt-badge ts" />
+你也可以看看：[为 ref 标注类型](/guide/typescript/composition-api.html#typing-ref) <sup class="vt-badge ts" />
 
-和响应式对象的属性类似，ref 的 `.value` 属性也是响应式的。同时，当值为对象类型时，会用 `reactive()` 自动转换它的 `.value`。
+和响应式对象的property类似，ref 的 `.value` property也是响应式的。同时，当值为对象类型时，会用 `reactive()` 自动转换它的 `.value`。
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:400]:
```suggestion
和响应式对象的 property 类似，ref 的 `.value` property 也是响应式的。同时，当值为对象类型时，会用 `reactive()` 自动转换它的 `.value`。
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 7]
```diff
@@ -451,9 +451,9 @@ function increment() {
 
 [在 Playground 尝试一下](https://sfc.vuejs.org/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgY291bnQgPSByZWYoMClcblxuZnVuY3Rpb24gaW5jcmVtZW50KCkge1xuICBjb3VudC52YWx1ZSsrXG59XG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8YnV0dG9uIEBjbGljaz1cImluY3JlbWVudFwiPnt7IGNvdW50IH19PC9idXR0b24+XG48L3RlbXBsYXRlPiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcInZ1ZVwiOiBcImh0dHBzOi8vc2ZjLnZ1ZWpzLm9yZy92dWUucnVudGltZS5lc20tYnJvd3Nlci5qc1wiXG4gIH1cbn0ifQ==)
 
-请注意，解包过程仅作用于顶层 property，访问深层级的 ref 则不会解包：
+请注意，模板渲染时解包只作用于 ref 是一个顶级property的情况。举个例子，`foo`是顶级property，但`object.foo`却不是。
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:454]:
```suggestion
请注意，模板渲染时解包只作用于 ref 是一个顶级 property 的情况。举个例子，`foo` 是顶级 property，但 `object.foo` 却不是。
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 8]
```diff
@@ -501,22 +501,22 @@ state.count = 1
 console.log(count.value) // 1
 ```
 
-如果将一个新的 ref 赋值给响应式对象某个已经为 ref 的属性，那么它会替换掉旧的 ref：
+如果将一个新的 ref 赋值给一个关联了已有 ref 的property，那么它会替换掉旧的 ref：
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:504]:
```suggestion
如果将一个新的 ref 赋值给一个关联了已有 ref 的 property，那么它会替换掉旧的 ref：
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 9]
```diff
@@ -501,22 +501,22 @@ state.count = 1
...
 
 state.count = otherCount
 console.log(state.count) // 2
-// 原来的 ref 现在已经和 state.count 脱去联系
+// 原始 ref 现在已经和 state.count 失去联系
 console.log(count.value) // 1
 ```
 
-只有当嵌套在一个深层反应式对象内时，才会发生 ref 解包。当起作为[浅层响应式对象](/api/reactivity-advanced.html#shallowreactive)的属性被访问时不会解包。
+只有当嵌套在一个深层响应式对象内时，才会发生 ref 解包。当起作为[浅层响应式对象](/api/reactivity-advanced.html#shallowreactive)的property被访问时不会解包。
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:515]:
```suggestion
只有当嵌套在一个深层响应式对象内时，才会发生 ref 解包。当起作为[浅层响应式对象](/api/reactivity-advanced.html#shallowreactive)的 property 被访问时不会解包。
```

File: src/guide/essentials/reactivity-fundamentals.md [hunk 10]
```diff
@@ -501,22 +501,22 @@ state.count = 1
...
 
 #### 数组和集合类型的 ref 解包 {#ref-unwrapping-in-arrays-and-collections}
 
-不像响应式对象，当从数组或 `Map` 这样的原生集合类型访问 ref 时，不会进行解包。
+不像响应式对象，当ref作为响应式数组或像`Map`这种原生集合类型的元素被访问时，不会进行解包。
```
Comment (SkyeYoung) [inline src/guide/essentials/reactivity-fundamentals.md:519]:
```suggestion
不像响应式对象，当 ref 作为响应式数组或像 `Map` 这种原生集合类型的元素被访问时，不会进行解包。
```

**Review by SkyeYoung — COMMENTED** [review]
Did you run `zhlint`?

**Comment by pakchoily** [comment]
I'll be careful,thanks for the reminder :sweat_smile:

> Did you run `zhlint`?

**Review by pakchoily — COMMENTED** [review]
Thanks for commits.

File: src/guide/essentials/reactivity-fundamentals.md [hunk 11]
```diff
@@ -465,7 +465,7 @@ const object = { foo: ref(1) }
 {{ object.foo + 1 }}
 ```
 
-渲染的结果会是一个 `[object Object]`，因为 `object.foo` 是一个 ref 对象。我们可以通过让 `foo` 成为顶层 property 来解决这个问题：
+渲染的结果会是一个 `[object Object]1`，因为 `object.foo` 是一个 ref 对象。我们可以通过让 `foo` 成为顶级 property 来解决这个问题：
```
[the remark this answers is not in the document]
Comment (pakchoily) [reply src/guide/essentials/reactivity-fundamentals.md:468]: 收到，确实已经改了。

File: src/guide/essentials/reactivity-fundamentals.md [hunk 12]
```diff
@@ -576,9 +576,9 @@ export default {
 
 <div class="composition-api">
 
-## 响应性语法糖 <sup class="vt-badge experimental" /> \*\* {#reactivity-transform}
```
[the remark this answers is not in the document]
Comment (pakchoily) [reply src/guide/essentials/reactivity-fundamentals.md:579]: 收到，我看英文版好像没有。

File: src/guide/essentials/reactivity-fundamentals.md [hunk 13]
```diff
@@ -576,9 +576,9 @@ export default {
...
+## 响应性语法糖 <sup class="vt-badge experimental" /> \*\*
```
Comment (wxsms) [inline src/guide/essentials/reactivity-fundamentals.md:579]:
```suggestion
## 响应性语法糖 <sup class="vt-badge experimental" /> \*\* {#reactivity-transform}
```

**Pull request merged**